recipes/drflac/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class DrflacConan(ConanFile):
+    name = "drflac"
+    description = "FLAC audio decoder."
+    homepage = "https://mackron.github.io/dr_flac"
+    topics = ("audio", "flac", "sound")
+    license = ("Unlicense", "MIT-0")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False],
+        "buffer_size": "ANY",
+        "no_crc": [True, False],
+        "no_ogg": [True, False],
+        "no_simd": [True, False],
+        "no_stdio": [True, False]
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+        "buffer_size": 0,
```

Comment:
I didn't want to hardcode the default buffer size here, that the library uses (it might also change between versions), thus using value zero to denote the default. Let me know if there is a better way.

Reply:
Please, add a comment here to denote that `0` means the default (not actual zero-sized buffer). It will help people reading this file.

Reply:
Done in abd0300405d13149debf6264d1b1cdcd37e4b889.